packages/core/src/compiler/index.ts

```diff
@@ -7,6 +7,10 @@ export interface AnalyzeOptions {
    * @default true
    */
   rerender: boolean
```

Comment:
We should consider renaming `rerender` to `rerenderTrace` for better readability.

Reply:
Resolved